Formula/crystal.rb

```diff
@@ -37,20 +36,14 @@ class Crystal < Formula
   depends_on "pcre"
 
   resource "boot" do
-    url "https://github.com/crystal-lang/crystal/releases/download/0.26.0/crystal-0.26.0-1-darwin-x86_64.tar.gz"
-    version "0.26.0-1"
-    sha256 "13ccd6425593f33f7423423553bc5c2fdcf5d76b6b97b82bf4204bc55831ec43"
+    url "https://github.com/crystal-lang/crystal/releases/download/0.26.1/crystal-0.26.1-1-darwin-x86_64.tar.gz"
+    version "0.26.1-1"
+    sha256 "3ad9616204d36ee4171e15892ee32216eab06f87f1f6cf5e32b45196dd4231d7"
   end
 
   def install
     (buildpath/"boot").install resource("boot")
 
-    if build.head?
-      ENV["CRYSTAL_CONFIG_VERSION"] = Utils.popen_read("git rev-parse --short HEAD").strip
-    else
-      ENV["CRYSTAL_CONFIG_VERSION"] = version
-    end
-
```

Comment:
Why should we remove these lines? If we build from HEAD, shouldn't we still set the version?

Reply:
The env var `CRYSTAL_CONFIG_VERSION` is no longer used. The version is fixed in the source code directly. For non tagged it will have a greater semver with `-dev` suffix.

I didn't consider using the formula to install head. In that case what it could be handy now is to set `CRYSTAL_CONFIG_BUILD_COMMIT` to `git rev-parse --short HEAD` as it is done directly in crystal repo.

Let me add it.